.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     # run every Sunday to re-populate caches after they are cleaned on Saturday
     - cron: "0 12 * * 0"
   push:
+    paths-ignore:
```

Comment:
We can use check job for API tests as well, no?

Reply:
API tests are running in submodules pipeline.  It was out of scope to move it here, also it will be problematic

Reply:
I mean before api-tests repo has its own CI jobs (linters, format, etc...). After this PR API tests code will not be covered by any CI, because of this `path-ignore`.  We can add `path-ignore` per job, for example we can cover API tests with `check` job, it checks licenses and runs linters. So there should be no problems.

Reply:
Look at https://github.com/Percona-Lab/pmm-api-tests/commits/master
there are no CI jobs right now.

Reply:
😨

Reply:
Ah, it was travisCI before(probably disabled for now): https://github.com/Percona-Lab/pmm-api-tests/pull/42

Well, IMHO we should include api-tests in CI flow, but maybe in separate ticket. I will create it.

Reply:
https://jira.percona.com/browse/PMM-8627